src/components/TopBar.js

```diff
@@ -32,6 +32,7 @@ export class TopBar extends Component {
       div(
         {
           style: {
+            zIndex: 7, // CodeMirror uses zIndexes, this seems to be enough.
```

Comment:
Please tell me there's a better way to do this.

Reply:
I would imagine the other option is to override CodeMirror's styles with sniper CSS.